README.md

```diff
@@ -42,9 +43,7 @@ npm install -g dohjs
 
 # Quickstart
 
-To get started hacking with dohjs, checkout our [examples](examples).
-
-Be sure to include doh.js in your HTML file. You can include it from [jsdelivr](https://www.jsdelivr.com/) or your local installation.
+First, be sure to include doh.js in your HTML file. You can include it from [jsdelivr](https://www.jsdelivr.com/) or your local installation.
 Make sure you put it before your other `<script>` tags.
```

Comment:
Does it need to be before? It's in the body for dohjs.org

Reply:
Mmm I guess it depends on how you're using it. See https://stackoverflow.com/questions/8996852/load-and-execute-order-of-scripts. I think I'll remove it for now.